Makefile

```diff
@@ -87,7 +87,7 @@ build:
 	make stop
 
 clean: deps
-	rm -rf $(GOPATH)/pkg/*
```

Comment:
well netplugin runs only on linux. But for development we would could be on mac, ubuntu,centos etc. clean would be run from ur ws. 
$ make clean
./scripts/deps
rm -rf /Users/aprativa//pkg/*
go clean -i -v ./...